lib/docker.js

```diff
@@ -171,7 +189,7 @@ async function getDockerUid(bindPath) {
     '/bin/sh',
   ];
   const ps = await dockerCommand(options);
-  return ps.stdout.trim();
+  return ps.stdoutBuffer.trim();
```

Comment:
I take it's a fix to previous PR, not related to modern logs?

Reply:
yes, I didn't spot it previously